airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/format_command.py

```diff
@@ -84,8 +85,10 @@ def get_dir_to_format(self, dagger_client: dagger.Client) -> dagger.Directory:
             Directory: The directory with the files to format that are not gitignored.
         """
         # Load a directory from the host with all the files to format according to the file_filter and the .gitignore files
+        modified_files = get_modified_files_local(get_current_git_revision())
+        self.logger.info(f"files to format: - {str(modified_files)}")
         dir_to_format = dagger_client.host().directory(
-            self.LOCAL_REPO_PATH, include=self.file_filter + ["**/.gitignore"], exclude=DEFAULT_FORMAT_IGNORE_LIST
```

Comment:
not sure why we were adding the .gitignore, only to delete them below.

Reply:
We're adding the gitignore so that the gitignored files are not formatted.